data/src/main/java/com/microsoft/azure/kusto/data/AadAuthenticationHelper.java

```diff
@@ -29,12 +29,14 @@
     private X509Certificate x509Certificate;
     private PrivateKey privateKey;
     private AuthenticationType authenticationType;
+    private String userToken;
```

Comment:
I think this should be renamed across all files to **accessToken**

Reply:
i used the variable name from our .Net library

Reply:
Unless I'm missing something I think that name isn't right on the other implementation.
Look at AadAuthenticationHelper.acquireAccessToken - userToken is actually returned and used as access token.
You can confirm with @toshetah to make sure.

Reply:
The userToken is defined in the connection string builder.
In the AadHelpers it is accessToken.